hangman.py

```diff
@@ -72,76 +69,91 @@ def get_guessed_word(secret_word, letters_guessed):
       which letters in secret_word have been guessed so far.
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-
+    formed_word = ""
+    for i in range(len(secret_word)):
+        if secret_word[i] in letters_guessed:
+            formed_word += secret_word[i]
+        else:
+            formed_word += HIDDEN_LETTER
 
+    return formed_word
 
 def get_available_letters(letters_guessed):
     '''
     letters_guessed: list (of letters), which letters have been guessed so far
     returns: string (of letters), comprised of letters that represents which letters have not
       yet been guessed.
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-    
-    
-
-def hangman(secret_word):
-    '''
-    secret_word: string, the secret word to guess.
-    
-    Starts up an interactive game of Hangman.
-    
-    * At the start of the game, let the user know how many 
-      letters the secret_word contains and how many guesses s/he starts with.
-      
-    * The user should start with 6 guesses
-
-    * Before each round, you should display to the user how many guesses
-      s/he has left and the letters that the user has not yet guessed.
-    
-    * Ask the user to supply one guess per round. Remember to make
-      sure that the user puts in a letter!
-    
-    * The user should receive feedback immediately after each guess 
-      about whether their guess appears in the computer's word.
-
-    * After each guess, you should display to the user the 
-      partially guessed word so far.
-    
-    Follows the other limitations detailed in the problem write-up.
-    '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
+    import string
+    x = list(string.ascii_lowercase)
+    if len(letters_guessed) == 0:
+        return ''.join(x)
+    else:
+        for symbol in letters_guessed:
+            del x[x.index(symbol.lower())]
+            y = ''.join(x)  # ''
+        return y
+
+
+def warnings(value_mistake, warnings_point, guesses_point, secret_word, letters_guessed):
+    """
+    Subtract warnings point and output name of error
+    """
+    if warnings_point > 0:
+        warnings_point -= 1
+        print(f"Oops! {value_mistake}", "\nYou have", warnings_point, "warnings left:", get_guessed_word(secret_word, letters_guessed))
+    else:
+        guesses_point -= 1
+        print("Oops! You have no warnings left, so you lose one guess:", get_guessed_word(secret_word, letters_guessed))
+    return warnings_point, guesses_point
+
+count_star = 0
+
+def print_hints_warning(secret_word, letters_guessed):
+    list_word = show_possible_matches(get_guessed_word(secret_word, letters_guessed))
+    if len(list_word) < 30:
 
+        print("Possible word matches are:", " ".join(list_word))
+    else:
+        random_words = []
+        for i in range(30):
+            random_words.append(choose_word(wordlist))
 
+        print("Possible word matches are:", " ".join(random_words))
```

Comment:
I expected to see it in Danilo's changes only, why you made some changes here? Danilo already merged his changes to develop, so you can re-base your branch on the top of develop, and you will have all code from Danilo